NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/LinkGroups.java

```diff
@@ -82,8 +82,6 @@ public boolean execute(CommandSender sender, String[] args) {
 		}
 		
 		boolean success = Group.link(supergroup, subgroup, true);
-		checkRecacheGroup(supergroup);
-		checkRecacheGroup(subgroup);
```

Comment:
Same comment as prior, but _especially_ here. Linking and unlinking was completely unuseable until I added recache.